qa/suites/fs/workload/tasks/3-snaps/yes.yaml

```diff
@@ -15,7 +15,8 @@ tasks:
       - ceph mgr module enable snap_schedule
       - ceph config set mgr mgr/snap_schedule/allow_m_granularity true
       - ceph config set mgr mgr/snap_schedule/dump_on_update true
-      - ceph fs snap-schedule add --fs=cephfs --path=/ --snap_schedule=1M
+      # Ensure snap_schedule module is loaded
+      - while ! ceph fs snap-schedule add --fs=cephfs --path=/ --snap_schedule=1M; do sleep 5; done
```

Comment:
Same here, I'd use `ceph fs snap-schedule status` in the body and move the loop directly after `ceph mgr module enable snap_schedule` so that the workaround is "attached" to the command.

And again, feel free to ignore this nit ;)